jinja_utils.py

```diff
@@ -61,6 +84,14 @@ def get_complete_static_resource_url(domain_url, resource_suffix):
         """Returns the relative path for the resource, appending it to the
         corresponding cache slug. resource_suffix should have a leading
         slash.
```

Comment:
You can drop the last sentence; it's already part of the args.

Reply:
Done